app/src/main/java/com/asuc/asucmobile/fragments/MapsFragment.java

```diff
@@ -335,8 +339,8 @@ public void onClick(View v) {
                 bundle.putString("Category", "microwaves");
```

Comment:
Is this the right Category here? Seems like it should be "waterbottles"

Reply:
Actually make it "Waterfountain" to match iOS styling so they are reported the same way in Firebase